requirements.txt

```diff
@@ -1,6 +1,5 @@
-aiida-core~=1.6.8
+aiida-core[atomic_tools]~=2.0.1
```

Comment:
As `atomic_tools` adds a lot of other packages, I'd prefer to have `pymatgen` be a specific, explicit package in this file and reverse this line to not include the `atomic_tools` extra.